### PR TITLE
Add optional query parameter to fetch live logs since the beginning

### PR DESCRIPTION
**Pull request by fviale**
also, add C# API link

### CODE REVIEW

File: dist/war/getstarted/index.html [hunk 1]
```diff
@@ -130,7 +130,10 @@ <h4 class="mt-5"><font size="+2">APIs</font> (REST, GraphQL, Javadoc) and <font
         <a id="PythonUrlLink" href="https://github.com/ow2-proactive/proactive-python-client#proactive-scheduler-client" target=”_blank”
            title="Python SDK">Python SDK</a>,
         <a id="javaUrlLink" href="https://doc.activeeon.com/javadoc/latest/org/ow2/proactive/scheduler/rest/SchedulerClient.html" target=”_blank”
-           title="JAVA SDK">JAVA SDK</a>.
+           title="JAVA SDK">JAVA SDK</a>
```
Comment (mboussaa) [inline dist/war/getstarted/index.html:133]: please add `,` after title="JAVA SDK">JAVA SDK</a>

Comment (fviale) [reply dist/war/getstarted/index.html:133]: thanks for spotting this !

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java [hunk 2]
```diff
@@ -356,21 +356,30 @@ public JobStateData listJobs(String sessionId, String jobId) throws RestExceptio
...
             Session session = sessionStore.get(sessionId);
 
             JobState jobState = scheduler.getJobState(jobId);
             boolean isFinished = jobState != null && jobState.isFinished();
-            int availableLinesCount = session.getJobsOutputController().availableLinesCount(jobId);
-
-            if (!isFinished || availableLinesCount > 0) {
-                return session.getJobsOutputController().getNewLogs(jobId);
+            if (allLogs) {
+                if (!isFinished) {
```
Comment (mboussaa) [inline rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java:368]: why `|| availableLinesCount > 0` could not be applied here?

**Review by mboussaa — APPROVED** [review]
Minor comments!

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java (hunk 2 above)
Comment (fviale) [reply rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java:368]: Ok, I will investigate

File: dist/war/getstarted/index.html [hunk 3]
```diff
@@ -130,7 +130,10 @@ <h4 class="mt-5"><font size="+2">APIs</font> (REST, GraphQL, Javadoc) and <font
         <a id="PythonUrlLink" href="https://github.com/ow2-proactive/proactive-python-client#proactive-scheduler-client" target=”_blank”
            title="Python SDK">Python SDK</a>,
         <a id="javaUrlLink" href="https://doc.activeeon.com/javadoc/latest/org/ow2/proactive/scheduler/rest/SchedulerClient.html" target=”_blank”
-           title="JAVA SDK">JAVA SDK</a>.
+           title="JAVA SDK">JAVA SDK</a>,
+        <a id="csharpUrlLink" href="https://www.activeeon.com/public_content/documentation/csharp-client/" target=”_blank”
```
Comment (amouhoub) [inline dist/war/getstarted/index.html:134]: Shouldn't the link documentation be included with the release and accessible locally ?

**Review by amouhoub — APPROVED** [review]

File: dist/war/getstarted/index.html (hunk 3 above)
Comment (fviale) [reply dist/war/getstarted/index.html:134]: there would be a lot of work to do that

Comment (amouhoub) [reply dist/war/getstarted/index.html:134]: That's what I thought.
You can merge. I will update the front-end

**Pull request merged**